packages/expo/src/Notifications/ExponentNotifications.web.ts

```diff
@@ -18,6 +20,19 @@ export default {
   async dismissAllNotifications(): Promise<void> {},
   async cancelScheduledNotification(notificationId: string): Promise<void> {},
   async cancelAllScheduledNotifications(): Promise<void> {},
-  //   getBadgeNumberAsync(): Promise<void> {},
-  //   setBadgeNumberAsync(badgeNumber: number): Promise<void> {},
+  async getBadgeNumberAsync(): Promise<number> {
+    return currentBadgeNumber;
+  },
+  async setBadgeNumberAsync(badgeNumber: number): Promise<void> {
+    currentBadgeNumber = badgeNumber;
+    
+    // Update title
+    const match = document.title.match(new RegExp('\(\d*\) (.*)'));
+    const originalTitle = match ? match[2] : document.title;
+    if (currentBadgeNumber) {
+      document.title = `(${currentBadgeNumber}) ${originalTitle}`;
```

Comment:
We may need to observe the document title as react-navigation updates it.

Reply:
I created a package https://www.npmjs.com/package/@approvals-cloud/badgin that uses the favicon first but if no favicon is available it proxies the title and also covers changes of the title.